chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc

```diff
@@ -433,5 +434,15 @@ void AddFootnoteViewIfNeeded(
   }                                                                       \
   AddGeolocationDescriptionIfNeeded(this, delegate_.get(), browser());
 
+// Sets the zorder for permission prompt bubble to kSecuritySurface, so that it
+// appears above other UI elements even they are floating on top.For example,
+// Picture-in-Picture window is on top of other widgets, but permission prompt
+// bubble should still be on top of it.
+#define SetZOrderSublevel(...)    \
+  SetZOrderSublevel(__VA_ARGS__); \
+  widget->SetZOrderLevel(ui::ZOrderLevel::kSecuritySurface);
+
 #include <chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc>
+
+#undef SetZOrderSublevel
```

Comment:
Would this introduce a scenario where only the permission prompt floats and becomes detached from the window it's related to or will it always elevate the prompt and the associated window?

Reply:
Only prompt bubble will be elevated. Do you think associated window should be brought to the top toghether?

Reply:
yeah, let me file a follow up issue